### ISSUE TITLE
[BUG] Login button is not visible when Brazilian Portuguese language is set

### ISSUE THREAD

**Issue by willemarcel** [hotosm/tasking-manager#6508]
**Describe the bug**
The Login button is not visible when Brazilian Portuguese language is set

**To Reproduce**
1. Switch language to pt-br

**Expected behavior**
Login button should be visible

**Screenshots**
![image](https://github.com/user-attachments/assets/0813cf22-a255-4973-aa06-cd838efec66b)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS
 - Browser [e.g. chrome, safari] Firefox

**Additional context**

**Comment by manjitapandey** [comment]
Hey @willemarcel can you confirm this issue still exist? I tried replicating on the same environment, but unsuccessful.

**Comment by willemarcel** [comment]
Yes, it exists. My screen width is 1440px. Maybe you're using a bigger screen...
